projects/cffi/decorated_fuzz_recompiler.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python3
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+import atheris
+
+from cffi import FFI, VerificationError, FFIError, CDefError
+from cffi import recompiler
+
+from py_io_capture import decorate_module, dump_records, DUMP_FILE_NAME
+import atexit
+cffi = decorate_module(cffi)
```

Comment:
decorate `recompiler`, actually for all `from module import xxx`, let's try decorate `xxx`.